### PR TITLE
feat(front): fixes related to calls

### PR DESCRIPTION
**Pull request by Skylsmoi**
https://github.com/tracim/tracim/issues/6402

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Comment by floreal15** [comment]
The refactor caused crash when the caller cancel its call

File: frontend/src/component/Call/Call.jsx [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    }
+  }
+
+  const handleClickOpenCallWindowCallee = () => {
+    props.dispatch(putSetIncomingUserCallState(props.user.userId, userCall.call_id, USER_CALL_STATE.ACCEPTED))
+    props.dispatch(setHeadTitle(props.system.headTitle))
+    audioCall.pause()
+  }
+
+  const handleClickRejectCall = () => {
```
Comment (floreal15) [inline frontend/src/component/Call/Call.jsx:124]: isn't it the same as handleUserCallModified, shouldn't only one handler be use for :
 - props.dispatch(setHeadTitle(props.system.headTitle))
 - audioCall.pause()

Comment (Skylsmoi) [reply frontend/src/component/Call/Call.jsx:124]: Indeed, will test it and will fix

**Comment by floreal15** [comment]
when someone make a call, the ringing last for around 27 sec before the call failed and ask the caller if he want to try again, in case he try again the ringing didn't have a maximum duration, as long as no one stop it it will continue ( it last for more than 10 minute before I stop it ) 
Should we have a fixed duration for the ringing ?
I got once a case when the first ringing stop after 27 second, the pop up disappeared but the music continue.

**Review by Millefeuille42 — APPROVED** [review]

**Comment by Millefeuille42** [comment]
tested on branch

**Pull request merged**